### DISCUSSION TITLE
Cloning socker_publisher project in Docker

### DISCUSSION THREAD

**Discussion by fljlk** [stella-cv/stella_vslam#521] [Q&A] [answered]
Hello (@ymd-stella and others)  !

I am trying to update stella_vslam in my docker image from 0.3.11 to 0.4.0 but I am left with the following error:

```cpp
Step 34/59 : RUN cd /tmp &&     git clone -b 0.0.1 --recursive https://github.com/stella-cv/socket_publisher.git && cd socket_publisher &&     mkdir build && cd build &&     cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo .. && make && make install

---> Running in 0545cd75a9b0
Cloning into 'socket_publisher'...

fatal: could not read Username for 'https://github.com': No such device or address

```
I would like to avoid creating Git users or ssh keys in my docker image.
I could clone every other repository without any problem, are there different permissions on the socket publisher one ?

**Comment by ymd-stella** [comment, marked as answer]
It seems that it was set to private. It is fixed now.

**Reply by fljlk** [reply]
Thank you so much !
